README.md

```diff
@@ -10,27 +10,25 @@ yarn install
 Install the dependencies.
 
 ```
-yarn build:api
+yarn build
 ```
 
-Run this to build `/dist/api` directory from the Ouroboros API, without building the entire site. It can be very slow on first run, as it fetches resources one at a time. API responses are cached in `.cache` for 30 days.
+Run this to run the following build scripts:
+1. `yarn build:assets` to build the `dist/assets/` directory for static site
+0. `yarn build:api` to build `dist/api/` directory from the Ouroboros API, without building the entire site. It can be very slow on first run, as it fetches resources one at a time. API responses are cached in `.cache/` for 30 days.
+0. `yarn build:site` to build static HTML pages in `dist/` for `users`, `tags`, `recents` and `collections` resources.
+It can be very slow on first run, as it fetches resources one at a time. API responses are cached in `.cache/` for 30 days.
+0. `yarn build:boards` to build the boards HTML pages in `dist/`.
+0. `yarn build:subjects` to build the subjects HTML pages in `dist/`.
```

Comment:
The boards and subjects scripts also build the `/api` JSON files for boards, discussions and subjects.